en/Project/index.md

```diff
@@ -28,6 +28,16 @@ The content managed within the project.
 - [InfoMotion](../InfoMotion/Introduction.md): Data visualization graphs dashboard
 - [InfoType](../InfoMotion/InfoTypeIntroduction.md): Graph types used with InfoMotion
 
+### Assets Searching
+
+You can serch the Assets associated with your account by using the searching box the top of the page.
```

Comment:
You can search the Assets associated with your account by using the search box at the the top of the page.